README.md

```diff
@@ -540,6 +540,9 @@ Calling `Item.persistence` returns an `ItemPersistence` object with the followin
   - .persistedState(timestamp, serviceId) ⇒ `PersistedItem | null`
   - .previousState(skipEqual, serviceId) ⇒ `PersistedItem | null`
   - .nextState(skipEqual, serviceId) ⇒ `PersistedItem | null`
+  - .riemannSumSince(timestamp, riemannType, serviceId)  ⇒ `PersistedState | null`: Time is considered in seconds.
```

Comment:
What is meant by this "Time is considered in seconds."?

Reply:
A Riemann sum will sum value * time by bucket. If the item type is a straight Number (no dimension, no unit), it is important to know the unit of the time dimension as the result will also be a straight Number. Calculation is always done in seconds for time.

Reply:
So if my Item is `Number:Power` with unit `kW` I will receive `kWs`?

Reply:
Yes, it will. But that gets converted automatically to kWh depending on the unit setting of the item if you persist, or the state description anyway.

Reply:
Do you think it would make sense to somehow improve the note in the README so the questions I asked won't come up if you just read this?